src/python/grpcio/grpc/experimental/aio/_interceptor.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Interceptors implementation of gRPC Asyncio Python."""
+from typing import Callable
+
+import grpc
```

Comment:
Where is the `_ClientCallDetails` defined?

Reply:
we reuse the one provided by the syncrhonous client.

Reply:
I'm gonna use for using a new fresh implementation provided by the `aio` module with a clone method, so removing it from the test